src/config/mockStrategy.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import passport from 'passport-strategy';
+import util from 'util';
```

Comment:
What is `util` used for?

Reply:
It is used in `line 19`:
 `util.inherits(Strategy, passport.Strategy);`.
It is making `Strategy` inherit `passport.Strategy`, by setting the prototype of `Strategy` to an instance of the prototype of `passport.Strategy`.

That way, we can pass `Strategy` as a middleware for `passport` to use and override the `authenticate` method that `Strategy` inherits with our own custom `authenticate` method.